docs/clients/step-functions.md

```diff
@@ -22,3 +22,45 @@ $result = $stepFunctions->startExecution(new StartExecutionInput([
 
 echo $result->getExecutionArn();
 ```
+
+### Send a task heartbeat
```

Comment:
No need to document all the methods we provide.
We don't pretend to replace the official AWS documentation.

Reply:
Cool. Maybe just success would have been enough. ;)